.github/workflows/lint-php.yml

```diff
@@ -0,0 +1,52 @@
+name: Lint PHP
+
+on:
+  workflow_call:
+    inputs:
+      PHP_MATRIX:
+        description: Matrix of PHP versions as a JSON formatted object.
+        default: '["8.0"]'
+        required: false
+        type: string
+      COMPOSER_ARGS:
+        description: Set of arguments passed to Composer.
+        default: '--prefer-dist'
+        required: false
+        type: string
+      LINT_ARGS:
+        description: Set of arguments passed to PHP Parallel Lint.
+        default: '. -e php --colors --show-deprecated'
+        required: false
+        type: string
+    secrets:
+      COMPOSER_AUTH_JSON:
+        description: Authentication for privately hosted packages and repositories as a JSON formatted object.
+        required: false
```

Comment:
See the other comment: we probably don't need to use Composer at all.

Reply:
https://github.com/inpsyde/reusable-workflows/pull/19#discussion_r995387999

Reply:
But if that's the case, it should be behind a input. For a single type of repository, which is the less common type of repository, we are going to affect significantly all the repositories that don't need it.

Reply:
Hi @gmazzap, unless I'm overlooking something, this is already behind an input (`COMPOSER_DEPS_INSTALL`), see https://github.com/inpsyde/reusable-workflows/pull/19/files#diff-931bfdfcfdadfc9685f2d544c3d0f533098b1b58e0f9fb070d747869971de4aaR21-R25, added in https://github.com/inpsyde/reusable-workflows/pull/19/files/5a2102d86f61a6c0ab12c400f4aaa1424748ad4c..78e97fac845a22db1cd113d48cef0fec22bc3919.